framework/wazuh/tests/test_util.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+import pytest
+import os
+
+from wazuh.utils import WazuhVersion
+
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+
+
+@pytest.mark.parametrize('version1, version2', [
```

Comment:
I miss a use case. WazuhVersion('v3.9.0') < WazuhVersion('v3.9.0') should return a false but it is not being tested in this unit tests. Same for WazuhVersion('v3.9.0') > WazuhVersion('v3.9.0')

Reply:
Done!